recipes/openmm-xtb/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "openmm-xtb" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/openmm-xtb/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 94778d570977f2f67df9330775c01b749bf6323cae76120067e39fc8601ed9f6
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - swig                                   # [build_platform != target_platform]
+    - openmm >=8.1.1                         # [build_platform != target_platform]
+    - cmake
+    - make
+    - conda-build  # [osx]
```

Comment:
This should never be listed as a dependency.
```suggestion
```

Reply:
Fixed.

Reply:
Actually, removing it broke the Mac build.

```
2024-03-06T01:30:42.7708550Z Traceback (most recent call last):
2024-03-06T01:30:42.7710240Z   File "/Users/runner/work/1/s/recipes/openmm-xtb/patch_osx_tests.py", line 4, in <module>
2024-03-06T01:30:42.7710700Z     from conda_build.os_utils import macho
2024-03-06T01:30:42.7711150Z ModuleNotFoundError: No module named 'conda_build'
```

Reply:
That error comes from the test helper script, where we had to patch some test binaries so they don't contain the hardcoded prefix. I'm not sure this is needed in this project (again, that was a plumed thing). I'd start simple with more standard approaches first, and then add hacks as needed using the existing openmm-* recipes as hints.

Reply:
As noted in my other comment, it really is needed.  I did exactly what you described: started with the simple approach, then copied over the fixes for problems from other recipes.